common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -714,54 +694,12 @@ def test_wiki_tab_first_time(self):
         )
         self.assertEqual(expected_article_name, course_wiki.article_name)
 
-    # TODO: TNL-6546: This whole function will be able to go away, replaced by test_course_home below.
-    def test_courseware_nav(self):
-        """
-        Navigate to a particular unit in the course.
-        """
-        # Navigate to the course page from the info page
-        self.course_info_page.visit()
-        self.tab_nav.go_to_tab('Course')
-
-        # Check that the course navigation appears correctly
```

Comment:
@dianakhuang: In general, we aren't making any changes that should make this test go away.  This tests that the legacy navigation from inside the courseware works.  If you think we have sufficient coverage in unit tests, maybe it is ok to do this in advance of killing this feature.  My concern about deleting tests too early is that it might be easy to break this code, and without this test, we would have no idea because we aren't using this ourselves.

Reply:
I think it's okay to rely on the unit tests here. We're trying to slim down our bok choy tests anyway.